Lib/aifc.py

```diff
@@ -371,7 +371,7 @@ def rewind(self):
         self._soundpos = 0
 
     def close(self):
-        file = self._file
+        file = getattr(self, '_file')
```

Comment:
It is easier and more reliable to set a class attribute `_file = None`. This also helps when `__del__` is called after clearing the `builtins` module. This is common idiom.